src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -278,17 +281,19 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
         emit OperatorSetParamsUpdated(quorumNumber, operatorSetParam);
     }
 
-    function _registerOperatorWithCoordinator(address operator, bytes calldata quorumNumbers, BN254.G1Point memory pubkey, string memory socket) internal {
-        require(
-            slasher.contractCanSlashOperatorUntilBlock(operator, address(serviceManager)) == type(uint32).max,
-            "StakeRegistry._registerOperator: operator must be opted into slashing by the serviceManager"
-        );
+    /// @return numOperatorsPerQuorum is the list of number of operators per quorum in quorumNumberss
+    function _registerOperatorWithCoordinator(address operator, bytes calldata quorumNumbers, BN254.G1Point memory pubkey, string memory socket) internal returns(uint32[] memory) {
+        // require(
+        //     slasher.contractCanSlashOperatorUntilBlock(operator, address(serviceManager)) == type(uint32).max,
+        //     "StakeRegistry._registerOperator: operator must be opted into slashing by the serviceManager"
+        // );
         
         // check that the sender is not already registered
         require(_operators[operator].status != OperatorStatus.REGISTERED, "BLSIndexRegistryCoordinator._registerOperatorWithCoordinator: operator already registered");
 
         // get the quorum bitmap from the quorum numbers
-        uint256 quorumBitmap = BitmapUtils.orderedBytesArrayToBitmap_Yul(quorumNumbers);
+        uint256 quorumBitmap = BitmapUtils.orderedBytesArrayToBitmap(quorumNumbers);
+
         require(quorumBitmap != 0, "BLSIndexRegistryCoordinator._registerOperatorWithCoordinator: quorumBitmap cannot be 0");
         require(quorumBitmap <= type(uint192).max, "BLSIndexRegistryCoordinator._registerOperatorWithCoordinator: quorumBitmap cant have more than 192 set bits");
```

Comment:
are these checks sufficient or should we also be checking something against the existing number of quorums? is that checked instead by the `stakeRegistry`? Just want to be confident we are adequately sanitizing the `quorumNumbers` input!

Reply:
I'll add that we can + should probably try to prove that you can't register for a non-existent quorum

Reply:
7815293c, this is done by the stake registry when it gets there